infrastructure/ci/dev-platform/shared/save-parameters-to-parameter-store.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -eu
+
+declare -r USAGE="Usage: $0 <stack-name>"
+
+if [[ "-h --help" =~ ( |^)${1:---help}( |$) ]]; then
+  echo "$USAGE"
+  exit 1
+fi
+
+if [[ "$#" -ne 1 ]]; then
+  echo "$USAGE"
+  exit 1
+fi
+
+BASE_DIR="$(dirname "${BASH_SOURCE[0]}")"
+pushd "$BASE_DIR" > /dev/null
```

Comment:
Do we need to `pushd` in this script? We're not accessing any relative files below.

If we do, I believe `popd > /dev/null` is missing at the end of the script

Reply:
👍🏻 we can do without it.